contents/tutorials/prevent-fouc-ab-tests.md

```diff
@@ -9,6 +9,8 @@ tags:
   - experimentation
 ---
 
+> 🚧 **NOTE:** As an alternative to the approach described below, we're experimenting with client-side feature flag assignments. Check out [https://posthog-fast-feature-flags.vercel.app/](https://posthog-fast-feature-flags.vercel.app/) to learn more. We are keen to gather as much feedback as possible so if you try this out please let us know. You can email [daniel.b@posthog.com](mailto:daniel.@posthog.com), send feedback via the [in-app support panel](https://us.posthog.com#panel=support%3Afeedback%3Aexperiments%3Alow), or use one of our other [support options](/docs/support-options).
```

Comment:
This is outside this PR but https://posthog-fast-feature-flags.vercel.app/ should mostly be a page in posthog.com/docs. We can link to it as a demo app. 

```suggestion
> 🚧 **Note:** As an alternative to the approach described below, we're experimenting with client-side feature flag assignments. Check out [https://posthog-fast-feature-flags.vercel.app/](https://posthog-fast-feature-flags.vercel.app/) to learn more. 
> We are keen to gather as much feedback as possible so if you try this out please let us know. You can email [daniel.b@posthog.com](mailto:daniel.@posthog.com), send feedback via the [in-app support panel](https://us.posthog.com#panel=support%3Afeedback%3Aexperiments%3Alow), or use one of our other [support options](/docs/support-options).
```